src/host/PtySignalInputThread.hpp

```diff
@@ -49,10 +51,15 @@ namespace Microsoft::Console
             unsigned short sx;
             unsigned short sy;
         };
+        struct SetParentData
+        {
+            uint64_t handle;
+        };

```

Comment:
Why's this a `struct`? Will there be more members in the future?
Why is `handle` not a `HWND`?

Reply:
Meh, they're all structs? I thought it was a nice abstraction to indicate "this is the packet of data that's sent". I don't expect to add more members in the future, but who knows.

As far as the `uint64_t` vs `HWND` - IIRC `HWND` is technically just a `void*`, so I wanted to be explicit with the size of data that's getting passed on the wire here. It's the same idea as like, every time we use a `uint64_t` for a `HWND` in any WinRT things.

Reply:
I think transmitting it as a `unintptr_t` would've been more consistent with our other Win32 APIs (which this API is closest to IMO).

Reply:
However, this is a wire format. 32-bit VSCode might be using 64-bit conhost (in fact: it _is_ using 64-bit conhost). `uintptr_t` changes size. `uint64_t` does not.